operator/v1/types_dns.go

```diff
@@ -287,6 +287,24 @@ type ForwardPlugin struct {
 	//
 	// +optional
 	TransportConfig DNSTransportConfig `json:"transportConfig,omitempty"`
+
+
+	// protocolStrategy specifies the protocol to use for upstream DNS
+	// requests. Valid values for protocolStrategy are ForceTCP or omitted.
```

Comment:
nit
```suggestion
	// requests. Valid values for protocolStrategy are "ForceTCP" or empty.
```

Reply:
Done. Also added `""` to every reference of "TCP"